test-harness.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+START=$(date "+%s")
+
+set -e
+
+ENV_FILE=".test-env"
+source $ENV_FILE
+
+echo "test-harness.sh: AFTER sourcing from $ENV_FILE. Build TYPE=$TYPE"
+
+rootdir=$(dirname "$0")
+pushd "$rootdir"
+
+## Reset test harness
+if [ -d "$SDK_TESTING_HARNESS" ]; then
+  pushd "$SDK_TESTING_HARNESS"
+  ./scripts/down.sh
+  popd
+  rm -rf "$SDK_TESTING_HARNESS"
+else
+  echo "test-harness.sh: directory $SDK_TESTING_HARNESS does not exist - NOOP"
+fi
+
+git clone --single-branch --branch "$SDK_TESTING_BRANCH" "$SDK_TESTING_URL" "$SDK_TESTING_HARNESS"
```

Comment:
@tzaffi Optionally, you may prefer to limit the cloned repo history as discussed in https://github.com/algorand/algorand-sdk-testing/pull/220#discussion_r946267733.  If you prefer no change, feel welcomed to resolve as is.

```suggestion
git clone --depth 1 --single-branch --branch "$SDK_TESTING_BRANCH" "$SDK_TESTING_URL" "$SDK_TESTING_HARNESS"
```